src/borg/helpers.py

```diff
@@ -1001,7 +1001,7 @@ def daemonize():
     os.close(0)
     os.close(1)
     os.close(2)
-    fd = os.open('/dev/null', os.O_RDWR)
+    fd = os.open(os.devnull, os.O_RDWR)
```

Comment:
Yeah, there's no real advantage, other than the fact that it's slightly nicer to use a predefined constant rather than hardcoding the string literal.